homeassistant/components/zha/core/channels/__init__.py

```diff
@@ -89,7 +89,13 @@ def __init__(self, cluster, device):
         self._generic_id = "channel_0x{:04x}".format(cluster.cluster_id)
         self._cluster = cluster
         self._zha_device = device
-        self._unique_id = construct_unique_id(cluster)
+        self._unique_id = "{}:{}:0x{:04x}".format(
```

Comment:
We can use f-strings now. Those are nicer.

Reply:
That's actually great news. Totally forgot since beta cut we can use f-strings.

Reply:
we'll update this to f-strings in subsequent PRs.